Pull from upstream repository, version 2.3.2

markbreedlove opened the pull request:
Pull commits from upstream `saltstack-formulas/nginx-formula`, tag `v2.2.1`.

Our changes to `nginx/ng/map.jinja` for upgrading the Nginx version have been abandoned.

I have moved our changes to `nginx/ng/config.sls`, to `nginx/config.sls`, with a minor alteration to correct the `source` of the `file.managed` state in `create_extra_config_{{ fname }}`.

pprkut committed:
Fix multiple values for the same key (include/listen) in server config file.

With this one can now also write

 - include:
   - file1
   - file2

Rather than only

 - include: file1
 - include: file2

Commit:
Fix build deps for NGINX NG build from source

francesco-a committed:
Disabled automatic sorting of mapping keys in jinja templating

ghtyrant committed:
Add support for snippets

amontalban committed:
Servers config should require available/enabled dirs

Commit:
[users.sls] touch /etc/nginx/.htpasswd

Commit:
[users] use {{ htauth }} variable

Commit:
[README] include nginx.ng.certifcates

javierbertoli committed:
Allow to specify a different passenger config file in nginx.ng

meganlkm committed:
implement test harness

Commit:
[nginx/users] add makedirs to htpasswd deploy

Commit:
[nginx/users] convert file.present into file.managed with no content

morsik committed:
Allow to use load_module without including external files

morsik committed:
Example for load_module

morsik committed:
Example for load_module: comment how it works

bebosudo committed:
Replace tabs in pillar.example with spaces.

tabs cause salt to trigger a render error because tabs are 'illegal':
```
yaml.scanner.ScannerError: while scanning for the next token
found character '\t' that cannot start any token
  in "<unicode string>", line 33, column 32:
            pid: /var/run/nginx.pid\t\t### Directory location must exist
                                   ^
```

MEschenbacher committed:
deploy certificates directly from pillar

... by providing a pillar string. I developed this for use in
combination with ext_pillar and file_tree to deploy letsencrypt
certificates.

aanriot committed:
Allow installation of multiple packages

ghtyrant committed:
Add example

bmwiedemann committed:
Use home and conf_dir from map.jinja

to reduce duplication of constants

bmwiedemann committed:
Add support for SUSE OSes

like openSUSE and SLES

bmwiedemann committed:
Fix detection of systemd

on openSUSE Leap 15.0, the string was systemd.cpython-36

noelmcloughlin committed:
makedirs is important for 'available_dir' pillar

noelmcloughlin committed:
Fix #182

noelmcloughlin committed:
Enforce PPA's on Ubuntu(s) only; not Debian

noelmcloughlin committed:
Document workaround for #122

noelmcloughlin committed:
Fix bug related to #188

noelmcloughlin committed:
Document issue 93 in pillar.example

noelmcloughlin committed:
Fully document #193 stream module

noelmcloughlin committed:
Document pid_file (#96)

noelmcloughlin committed:
Update based on review comments

noelmcloughlin committed:
Updated for Suse 15.0

Commit:
added default snippets directory for FreeBSD platform

daks committed:
fix: corrected indentation on context in file.managed states

garrettw committed:
Pillar fixes (#217)

Squashed from:

* pillar.example: an include was missing a colon
* pillar.example: 2 directives not indented properly
* pillar.example: added the stock log_format
* pillar.example: make log path match map.jinja
* pillar.example: worker_connections value from latest mainline nginx
* pillar.example: fixed other uses of include
* pillar.example: corrected formatting
* pillar.example: corrected log_format
* pillar.example: reset log_format to try another day

daks committed:
test(inspec): switch tests to use Inspec

daks committed:
ci(travis): update test matrix

daks committed:
ci(travis): disabling CentOS7 tests for now

daks committed:
test(inspec): more tests on nginx config

daks committed:
docs(README): remove obsolete tests section

myii committed:
fix(gemfile): use latest `kitchen-salt`

myii committed:
ci(travis): fix `opensuse` tests

myii committed:
ci(kitchen+travis): use `py2` instead of `py3` for `centos-7`

* https://github.com/saltstack-formulas/salt-formula/pull/414

daks committed:
docs(semantic-release): implement an automated changelog

daks committed:
WIP

semantic-release-bot committed:
chore(release): 0.55.1 [skip ci]

## [0.55.1](https://github.com/saltstack-formulas/nginx-formula/compare/v0.55.0...v0.55.1) (2019-04-26)

### Documentation

* **semantic-release:** implement an automated changelog ([569b07a](https://github.com/saltstack-formulas/nginx-formula/commit/569b07a))

myii committed:
test(inspec): add test for `log_format` #219

Commit:
feat(`pillar.example`): add stock `log_format`

semantic-release-bot committed:
chore(release): 0.56.0 [skip ci]

# [0.56.0](https://github.com/saltstack-formulas/nginx-formula/compare/v0.55.1...v0.56.0) (2019-04-26)

### Features

* **`pillar.example`:** add stock `log_format` ([95ff308](https://github.com/saltstack-formulas/nginx-formula/commit/95ff308))

semantic-release-bot committed:
chore(release): 0.56.1 [skip ci]

## [0.56.1](https://github.com/saltstack-formulas/nginx-formula/compare/v0.56.0...v0.56.1) (2019-04-27)

### Tests

* **inspec:** add test for `log_format` [#219](https://github.com/saltstack-formulas/nginx-formula/issues/219) ([4ed788e](https://github.com/saltstack-formulas/nginx-formula/commit/4ed788e))

daks committed:
refactor: replace old `nginx` with `nginx.ng`

BREAKING CHANGE: all previous `nginx` based configurations must be reviewed;
`nginx.ng` usage must be promoted to `nginx` and any uses of the original
`nginx` will have to be converted.

daks committed:
build: remove obsolete Makefile

semantic-release-bot committed:
chore(release): 1.0.0 [skip ci]

# [1.0.0](https://github.com/saltstack-formulas/nginx-formula/compare/v0.56.1...v1.0.0) (2019-05-12)

### Build System

* remove obsolete Makefile ([4961b04](https://github.com/saltstack-formulas/nginx-formula/commit/4961b04))

### Code Refactoring

* replace old `nginx` with `nginx.ng` ([0fc5070](https://github.com/saltstack-formulas/nginx-formula/commit/0fc5070))

### BREAKING CHANGES

* all previous `nginx` based configurations must be reviewed;
`nginx.ng` usage must be promoted to `nginx` and any uses of the original
`nginx` will have to be converted.

noelmcloughlin committed:
docs: remove obsolete CHANGELOG.rst file

semantic-release-bot committed:
chore(release): 1.0.1 [skip ci]

## [1.0.1](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.0...v1.0.1) (2019-05-12)

### Documentation

* remove obsolete CHANGELOG.rst file ([698aadb](https://github.com/saltstack-formulas/nginx-formula/commit/698aadb))

n-rodriguez committed:
docs(readme): update README, add badges

semantic-release-bot committed:
chore(release): 1.0.2 [skip ci]

## [1.0.2](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.1...v1.0.2) (2019-05-12)

### Documentation

* **readme:** update README, add badges ([adbac43](https://github.com/saltstack-formulas/nginx-formula/commit/adbac43))

myii committed:
docs(readme): add warning banner about `v1.0.0` breaking changes

semantic-release-bot committed:
chore(release): 1.0.3 [skip ci]

## [1.0.3](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.2...v1.0.3) (2019-05-13)

### Documentation

* **readme:** add warning banner about `v1.0.0` breaking changes ([d553821](https://github.com/saltstack-formulas/nginx-formula/commit/d553821))

myii committed:
fix: prevent running of states deprecated in `v1.0.0`

* Route all to `nginx/deprecated.sls`

semantic-release-bot committed:
chore(release): 1.0.4 [skip ci]

## [1.0.4](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.3...v1.0.4) (2019-05-13)

### Bug Fixes

* prevent running of states deprecated in `v1.0.0` ([46dff15](https://github.com/saltstack-formulas/nginx-formula/commit/46dff15))

n-rodriguez committed:
docs(readme): improve readme sections

semantic-release-bot committed:
chore(release): 1.0.5 [skip ci]

## [1.0.5](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.4...v1.0.5) (2019-05-13)

### Documentation

* **readme:** improve readme sections ([3cc3407](https://github.com/saltstack-formulas/nginx-formula/commit/3cc3407))

Commit:
feat(tofs): first implemetation + tplroot

First implementation of TOFS for:
nginx_config, passenger_config, servers_config and snippets
Introduced tplroot on modified files as well
Fixed GH link of nginx.conf file in docs/TOFS_pattern.rst
Fixed test for snippets name pillar

semantic-release-bot committed:
chore(release): 1.1.0 [skip ci]

# [1.1.0](https://github.com/saltstack-formulas/nginx-formula/compare/v1.0.5...v1.1.0) (2019-06-03)

### Features

* **tofs:** first implemetation + tplroot ([d5262ea](https://github.com/saltstack-formulas/nginx-formula/commit/d5262ea))

Commit:
fix(snippets): removed appending of ".conf"

(...) for snippets filename.
BREAKING CHANGE: Users have to modify their pillar
according to this commit. Users MUST append '.conf' for their
existing managed snippets.

semantic-release-bot committed:
chore(release): 2.0.0 [skip ci]

# [2.0.0](https://github.com/saltstack-formulas/nginx-formula/compare/v1.1.0...v2.0.0) (2019-06-19)

### Bug Fixes

* **snippets:** removed appending of ".conf" ([aa87721](https://github.com/saltstack-formulas/nginx-formula/commit/aa87721))

### BREAKING CHANGES

* **snippets:** Users have to modify their pillar
according to this commit. Users MUST append '.conf' for their
existing managed snippets.

myii committed:
ci(kitchen+travis): modify matrix to include `develop` platform

* Use balanced matrix based on `template-formula` guidelines

myii committed:
chore: use `semantic-release` cross-formula standard structure

* Automated using `ssf-formula` (v0.1.0-rc.1)

ErisDS committed:
tofs: include path pattern with defaults in docs

ErisDS committed:
feat(passenger): inc config, snippets, servers, etc

- currently you're forced to define extra states if you opt to install nginx with passenger,
 if you want the same outcome for passenger and nginx installs with equivalent config
- passenger is an extra module on top of nginx, makes no sense to end up with less configuration by default

myii committed:
chore: use `semantic-release` cross-formula standard structure

* Automated using `ssf-formula` (v0.1.0-rc.2)

noelmcloughlin committed:
feat(linux): archlinux support (no osfinger grain)

semantic-release-bot committed:
chore(release): 2.1.0 [skip ci]

# [2.1.0](https://github.com/saltstack-formulas/nginx-formula/compare/v2.0.0...v2.1.0) (2019-08-04)

### Continuous Integration

* **kitchen+travis:** modify matrix to include `develop` platform ([f6b357d](https://github.com/saltstack-formulas/nginx-formula/commit/f6b357d))

### Features

* **linux:** archlinux support (no osfinger grain) ([ab6148c](https://github.com/saltstack-formulas/nginx-formula/commit/ab6148c))

myii committed:
feat(yamllint): include for this repo and apply rules throughout

* Semi-automated using `ssf-formula` (v0.5.0)
* Fix errors shown below:

```bash
nginx-formula$ $(grep "\- yamllint" .travis.yml | sed -e "s:^\s\+-\s\(.*\):\1:")
pillar.example
  5:1       warning  missing document start "---"  (document-start)
  6:89      error    line too long (110 > 88 characters)  (line-length)
  7:89      error    line too long (99 > 88 characters)  (line-length)
  19:89     error    line too long (132 > 88 characters)  (line-length)
  35:25     warning  truthy value should be one of [false, true]  (truthy)
  36:89     error    line too long (106 > 88 characters)  (line-length)
  40:89     error    line too long (115 > 88 characters)  (line-length)
  43:16     warning  truthy value should be one of [false, true]  (truthy)
  44:89     error    line too long (105 > 88 characters)  (line-length)
  48:16     warning  truthy value should be one of [false, true]  (truthy)
  54:14     warning  too few spaces before comment  (comments)
  57:13     warning  truthy value should be one of [false, true]  (truthy)
  57:18     warning  too few spaces before comment  (comments)
  58:14     warning  too few spaces before comment  (comments)
  60:5      warning  missing starting space in comment  (comments)
  60:89     error    line too long (96 > 88 characters)  (line-length)
  61:89     error    line too long (90 > 88 characters)  (line-length)
  62:89     error    line too long (90 > 88 characters)  (line-length)
  63:89     error    line too long (94 > 88 characters)  (line-length)
  64:5      warning  missing starting space in comment  (comments)
  64:89     error    line too long (96 > 88 characters)  (line-length)
  68:9      error    wrong indentation: expected 10 but found 8  (indentation)
  76:9      error    wrong indentation: expected 10 but found 8  (indentation)
  86:9      error    wrong indentation: expected 10 but found 8  (indentation)
  90:14     warning  too few spaces before comment  (comments)
  90:89     error    line too long (104 > 88 characters)  (line-length)
  92:6      warning  missing starting space in comment  (comments)
  92:89     error    line too long (106 > 88 characters)  (line-length)
  94:89     error    line too long (104 > 88 characters)  (line-length)
  97:6      warning  missing starting space in comment  (comments)
  97:89     error    line too long (106 > 88 characters)  (line-length)
  100:62    warning  too few spaces before comment  (comments)
  100:89    error    line too long (121 > 88 characters)  (line-length)
  101:62    warning  comment not indented like content  (comments-indentation)
  101:89    error    line too long (116 > 88 characters)  (line-length)
  102:89    error    line too long (112 > 88 characters)  (line-length)
  103:89    error    line too long (99 > 88 characters)  (line-length)
  105:89    error    line too long (124 > 88 characters)  (line-length)
  106:8     warning  missing starting space in comment  (comments)
  106:89    error    line too long (117 > 88 characters)  (line-length)
  112:89    error    line too long (97 > 88 characters)  (line-length)
  120:31    warning  missing starting space in comment  (comments)
  123:89    error    line too long (95 > 88 characters)  (line-length)
  127:13    error    wrong indentation: expected 14 but found 12  (indentation)
  151:33    warning  too few spaces before comment  (comments)
  151:89    error    line too long (94 > 88 characters)  (line-length)
  152:22    warning  too few spaces before comment  (comments)
  152:89    error    line too long (90 > 88 characters)  (line-length)
  153:21    warning  too few spaces before comment  (comments)
  153:89    error    line too long (101 > 88 characters)  (line-length)
  154:22    warning  too few spaces before comment  (comments)
  155:18    warning  too few spaces before comment  (comments)
  155:89    error    line too long (103 > 88 characters)  (line-length)
  163:89    error    line too long (100 > 88 characters)  (line-length)
  164:89    error    line too long (106 > 88 characters)  (line-length)
  165:18    warning  truthy value should be one of [false, true]  (truthy)
  167:89    error    line too long (109 > 88 characters)  (line-length)
  170:10    warning  missing starting space in comment  (comments)
  172:10    warning  missing starting space in comment  (comments)
  172:89    error    line too long (120 > 88 characters)  (line-length)
  173:10    warning  missing starting space in comment  (comments)
  173:89    error    line too long (118 > 88 characters)  (line-length)
  174:89    error    line too long (120 > 88 characters)  (line-length)
  175:20    warning  truthy value should be one of [false, true]  (truthy)
  175:89    error    line too long (102 > 88 characters)  (line-length)
  177:89    error    line too long (100 > 88 characters)  (line-length)
  178:89    error    line too long (101 > 88 characters)  (line-length)
  182:13    error    wrong indentation: expected 14 but found 12  (indentation)
  184:15    error    wrong indentation: expected 16 but found 14  (indentation)
  186:15    error    wrong indentation: expected 16 but found 14  (indentation)
  189:15    error    wrong indentation: expected 16 but found 14  (indentation)
  193:9     warning  comment not indented like content  (comments-indentation)
  196:13    error    wrong indentation: expected 14 but found 12  (indentation)
  198:15    error    wrong indentation: expected 16 but found 14  (indentation)
  202:15    error    wrong indentation: expected 16 but found 14  (indentation)
  206:9     warning  comment not indented like content  (comments-indentation)
  218:16    warning  too few spaces before comment  (comments)
  218:89    error    line too long (96 > 88 characters)  (line-length)
  219:18    warning  truthy value should be one of [false, true]  (truthy)
  223:57    warning  too few spaces before comment  (comments)
  223:89    error    line too long (111 > 88 characters)  (line-length)
  224:57    warning  comment not indented like content  (comments-indentation)
  224:89    error    line too long (118 > 88 characters)  (line-length)
  226:89    error    line too long (90 > 88 characters)  (line-length)
  236:18    warning  truthy value should be one of [false, true]  (truthy)
  245:89    error    line too long (108 > 88 characters)  (line-length)
  252:89    error    line too long (104 > 88 characters)  (line-length)
  305:89    error    line too long (115 > 88 characters)  (line-length)
  306:32    error    trailing spaces  (trailing-spaces)

test/salt/default/pillar/nginx.sls
  1:1       error    too many blank lines (1 > 0)  (empty-lines)
  7:1       warning  missing document start "---"  (document-start)
  11:9      error    wrong indentation: expected 10 but found 8  (indentation)
  23:18     warning  truthy value should be one of [false, true]  (truthy)
  24:18     warning  truthy value should be one of [false, true]  (truthy)
  28:18     warning  truthy value should be one of [false, true]  (truthy)
  31:13     error    wrong indentation: expected 14 but found 12  (indentation)
  33:15     error    wrong indentation: expected 16 but found 14  (indentation)
  36:15     error    wrong indentation: expected 16 but found 14  (indentation)
  38:1      error    too many blank lines (1 > 0)  (empty-lines)
```

semantic-release-bot committed:
chore(release): 2.2.0 [skip ci]

# [2.2.0](https://github.com/saltstack-formulas/nginx-formula/compare/v2.1.0...v2.2.0) (2019-08-12)

### Features

* **yamllint:** include for this repo and apply rules throughout ([6b7d1fe](https://github.com/saltstack-formulas/nginx-formula/commit/6b7d1fe))

myii committed:
chore: use `semantic-release` cross-formula standard structure

* Automated using `ssf-formula` (v0.6.0)

n-rodriguez committed:
docs(readme): update testing section

semantic-release-bot committed:
chore(release): 2.2.1 [skip ci]

## [2.2.1](https://github.com/saltstack-formulas/nginx-formula/compare/v2.2.0...v2.2.1) (2019-08-25)

### Documentation

* **readme:** update testing section ([182f216](https://github.com/saltstack-formulas/nginx-formula/commit/182f216))

myii committed:
ci(kitchen+travis): replace EOL pre-salted images

* Automated using https://github.com/myii/ssf-formula/pull/16

markbreedlove commented on nginx/config.sls:
```diff
@@ -0,0 +1,45 @@
+    - makedirs: True
+    - context:
+        config: {{ settings|json(sort_keys=False) }}
+{% endfor %}
```
The block above is the one that I copied from `nginx/ng/config.sls` and modified slightly to fix the `source` attribute of `file.managed`.

semantic-release-bot committed:
chore(release): 2.3.0 [skip ci]

# [2.3.0](https://github.com/saltstack-formulas/nginx-formula/compare/v2.2.1...v2.3.0) (2019-09-01)

### Continuous Integration

* **kitchen+travis:** replace EOL pre-salted images ([70e1426](https://github.com/saltstack-formulas/nginx-formula/commit/70e1426))

### Features

* **passenger:** inc config, snippets, servers, etc ([e07b558](https://github.com/saltstack-formulas/nginx-formula/commit/e07b558))

myii committed:
ci: use `dist: bionic` & apply `opensuse-leap-15` SCP error workaround

* Automated using https://github.com/myii/ssf-formula/pull/22

myii committed:
ci(yamllint): add rule `empty-values` & use new `yaml-files` setting

* Semi-automated using https://github.com/myii/ssf-formula/pull/27
* Fix errors shown below:

```bash
nginx-formula$ yamllint -s .
./pillar.example
  146:56    error    empty value in block mapping  (empty-values)
  147:76    error    empty value in block mapping  (empty-values)
  148:59    error    empty value in block mapping  (empty-values)
  150:22    error    empty value in block mapping  (empty-values)
  151:39    error    empty value in block mapping  (empty-values)
  152:39    error    empty value in block mapping  (empty-values)
  153:39    error    empty value in block mapping  (empty-values)
```

blarghmatey commented:
Looks good, but before we merge we'll need to update our pillar data accordingly. Might want to wait until we get the separate masters set up so we can pin prod and non-prod separately.

myii committed:
ci(platform): add `arch-base-latest`

* Automated using https://github.com/myii/ssf-formula/pull/40

myii committed:
ci(kitchen): change `log_level` to `debug` instead of `info`

* Automated using https://github.com/myii/ssf-formula/pull/41

myii committed:
chore(issues): provide `Bug report` & `Feature request` templates [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/48

myii committed:
ci(kitchen): use bootstrapped `opensuse` images until `2019.2.2` [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/52

myii committed:
ci(kitchen): install required packages to bootstrapped `opensuse` [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/53

myii committed:
docs(pillar.example): fix TOFS comment to explain the default path [skip ci]

* Replace the Jinja brackets to avoid evaluating when `pillar.example` is
  used from Kitchen
* References:
  - https://github.com/saltstack-formulas/libvirt-formula/pull/60#issuecomment-537965254
  - https://github.com/saltstack-formulas/libvirt-formula/pull/60#issuecomment-537988138

myii committed:
chore(issues): update `Bug report` & `Feature request` templates [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/58

myii committed:
chore(yamllint): update ignored paths and add `octal-values` [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/59

myii committed:
ci: merge travis matrix, add `salt-lint` & `rubocop` to `lint` job

* Automated using https://github.com/myii/ssf-formula/pull/60

myii committed:
fix(certificates.sls): fix `salt-lint` errors

```bash
Examining nginx/certificates.sls of type state
[206] Jinja variables should have spaces before and after: {{ var_name }}
nginx/certificates.sls:40
    - contents_pillar: {{salt['pillar.get']('nginx:certificates:{}:public_cert_pillar'.format(domain))}}

[202] Jinja statement should have spaces before and after: {% statement %}
nginx/certificates.sls:47
{% if salt['pillar.get']("nginx:certificates:{}:private_key".format(domain)) or salt['pillar.get']("nginx:certificates:{}:private_key_pillar".format(domain))%}

[206] Jinja variables should have spaces before and after: {{ var_name }}
nginx/certificates.sls:54
    - contents_pillar: {{salt['pillar.get']('nginx:certificates:{}:private_key_pillar'.format(domain))}}
```

myii committed:
fix(map.jinja): fix `salt-lint` errors

```bash
Examining nginx/map.jinja of type state
[202] Jinja statement should have spaces before and after: {% statement %}
nginx/map.jinja:175
})%}

[202] Jinja statement should have spaces before and after: {% statement %}
nginx/map.jinja:181
})%}

[202] Jinja statement should have spaces before and after: {% statement %}
nginx/map.jinja:188
})%}

[202] Jinja statement should have spaces before and after: {% statement %}
nginx/map.jinja:192
    })%}

[202] Jinja statement should have spaces before and after: {% statement %}
nginx/map.jinja:195
    })%}
```

myii committed:
fix(pkg.sls): fix `salt-lint` errors

```bash
Examining nginx/pkg.sls of type state
[201] Trailing whitespace
nginx/pkg.sls:149
    - gpgcheck: 0
```

myii committed:
ci: merge travis matrix, add `salt-lint` & `rubocop` to `lint` job

* Semi-automated using https://github.com/myii/ssf-formula/pull/60

semantic-release-bot committed:
chore(release): 2.3.1 [skip ci]

## [2.3.1](https://github.com/saltstack-formulas/nginx-formula/compare/v2.3.0...v2.3.1) (2019-10-10)

### Bug Fixes

* **certificates.sls:** fix `salt-lint` errors ([](https://github.com/saltstack-formulas/nginx-formula/commit/bedc1b6))
* **map.jinja:** fix `salt-lint` errors ([](https://github.com/saltstack-formulas/nginx-formula/commit/0772d8a))
* **pkg.sls:** fix `salt-lint` errors ([](https://github.com/saltstack-formulas/nginx-formula/commit/06d055e))

### Continuous Integration

* **kitchen:** change `log_level` to `debug` instead of `info` ([](https://github.com/saltstack-formulas/nginx-formula/commit/671a4ce))
* **kitchen:** install required packages to bootstrapped `opensuse` [skip ci] ([](https://github.com/saltstack-formulas/nginx-formula/commit/17291a0))
* **kitchen:** use bootstrapped `opensuse` images until `2019.2.2` [skip ci] ([](https://github.com/saltstack-formulas/nginx-formula/commit/a39e124))
* **platform:** add `arch-base-latest` ([](https://github.com/saltstack-formulas/nginx-formula/commit/c921086))
* **yamllint:** add rule `empty-values` & use new `yaml-files` setting ([](https://github.com/saltstack-formulas/nginx-formula/commit/3d48b1b))
* merge travis matrix, add `salt-lint` & `rubocop` to `lint` job ([](https://github.com/saltstack-formulas/nginx-formula/commit/08ce3ed))
* use `dist: bionic` & apply `opensuse-leap-15` SCP error workaround ([](https://github.com/saltstack-formulas/nginx-formula/commit/8ddb921))

### Documentation

* **pillar.example:** fix TOFS comment to explain the default path [skip ci] ([](https://github.com/saltstack-formulas/nginx-formula/commit/714f547)), closes [/github.com/saltstack-formulas/libvirt-formula/pull/60#issuecomment-537965254](https://github.com//github.com/saltstack-formulas/libvirt-formula/pull/60/issues/issuecomment-537965254) [/github.com/saltstack-formulas/libvirt-formula/pull/60#issuecomment-537988138](https://github.com//github.com/saltstack-formulas/libvirt-formula/pull/60/issues/issuecomment-537988138)

myii committed:
docs(readme): update link to `CONTRIBUTING` [skip ci]

myii committed:
docs(contributing): remove to use org-level file instead [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/70

myii committed:
ci(travis): update `salt-lint` config for `v0.0.10` [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/82

myii committed:
ci(kitchen): use `debian-10-master-py3` instead of `develop` [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/84

myii committed:
ci(kitchen+travis): upgrade matrix after `2019.2.2` release [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/86

myii committed:
fix(release.config.js): use full commit hash in commit link [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/89

myii committed:
perf(travis): improve `salt-lint` invocation [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/90

myii committed:
ci(kitchen): use `develop` image until `master` is ready (`amazonlinux`) [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/93

myii committed:
ci(travis): use build config validation (beta) [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/97

myii committed:
ci(travis): apply changes from build config validation [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/98

myii committed:
ci(travis): opt-in to `dpl v2` to complete build config validation [skip ci]

* Automated using https://github.com/myii/ssf-formula/pull/100

myii committed:
test(pillar/nginx.sls): add reprodicible snippet based on issue #241

myii committed:
fix(certificates.sls): prepare `certificates_path` dir separately

* Close #241

semantic-release-bot committed:
chore(release): 2.3.2 [skip ci]

## [2.3.2](https://github.com/saltstack-formulas/nginx-formula/compare/v2.3.1...v2.3.2) (2019-11-25)

### Bug Fixes

* **certificates.sls:** prepare `certificates_path` dir separately ([297e3ac](https://github.com/saltstack-formulas/nginx-formula/commit/297e3ac400707cdd8f396da4c23ba30fc719a2cd)), closes [#241](https://github.com/saltstack-formulas/nginx-formula/issues/241)
* **release.config.js:** use full commit hash in commit link [skip ci] ([b13ec85](https://github.com/saltstack-formulas/nginx-formula/commit/b13ec85433d85b8ca87c3798db9cab3e297b81cf))

### Continuous Integration

* **kitchen:** use `debian-10-master-py3` instead of `develop` [skip ci] ([0665878](https://github.com/saltstack-formulas/nginx-formula/commit/066587829c5a40967b0e7926f12202b07b51ab3c))
* **kitchen:** use `develop` image until `master` is ready (`amazonlinux`) [skip ci] ([e8ed39a](https://github.com/saltstack-formulas/nginx-formula/commit/e8ed39a62cd40fe43af2aae67a3e2347d02b6b6a))
* **kitchen+travis:** upgrade matrix after `2019.2.2` release [skip ci] ([faefcab](https://github.com/saltstack-formulas/nginx-formula/commit/faefcabd654e5323b6ca146fb0046dd636ed5f68))
* **travis:** apply changes from build config validation [skip ci] ([4125887](https://github.com/saltstack-formulas/nginx-formula/commit/41258874a52df3da7a9f036b5378eb12b7a1a537))
* **travis:** opt-in to `dpl v2` to complete build config validation [skip ci] ([dbeb2da](https://github.com/saltstack-formulas/nginx-formula/commit/dbeb2da3e43aa13f162b1ac4c6203ecff60e0102))
* **travis:** update `salt-lint` config for `v0.0.10` [skip ci] ([a8382b5](https://github.com/saltstack-formulas/nginx-formula/commit/a8382b51a028ed5f069ff0168127ef3c8a4337da))
* **travis:** use build config validation (beta) [skip ci] ([bbf91c9](https://github.com/saltstack-formulas/nginx-formula/commit/bbf91c9f1432118a9eafde507de9ffa7b3ff5093))
* merge travis matrix, add `salt-lint` & `rubocop` to `lint` job ([567c08c](https://github.com/saltstack-formulas/nginx-formula/commit/567c08c9adf752eb95627b0e914804645015ee20))

### Documentation

* **contributing:** remove to use org-level file instead [skip ci] ([2e58d63](https://github.com/saltstack-formulas/nginx-formula/commit/2e58d636aaa8a66ec9540238b2f4e267172e10c2))
* **readme:** update link to `CONTRIBUTING` [skip ci] ([3ff6692](https://github.com/saltstack-formulas/nginx-formula/commit/3ff6692590932e7cc7609fdc0f52fc261228f290))

### Performance Improvements

* **travis:** improve `salt-lint` invocation [skip ci] ([e586fbe](https://github.com/saltstack-formulas/nginx-formula/commit/e586fbeebc758cdfd6d381a6ef9ad72231523dea))

### Tests

* **pillar/nginx.sls:** add reprodicible snippet based on issue [#241](https://github.com/saltstack-formulas/nginx-formula/issues/241) ([4ba3524](https://github.com/saltstack-formulas/nginx-formula/commit/4ba35247ed742393367968db34ff61a6b07f6695))

markbreedlove commented:
I pulled more changes from the upstream, so now we're looking at version 2.3.2 of `saltstack-formulas/nginx-formula`.

blarghmatey commented:
:+1:
